f5-bigip-mixin/dashboards/bigip-virtual-server-overview.libsonnet

```diff
@@ -0,0 +1,618 @@
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [],
+      displayMode: 'list',
+      placement: 'bottom',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local trafficPanel = {
+  datasource: promDatasource,
+  targets: [
+    prometheus.target(
+      'rate(bigip_vs_clientside_bytes_in{job=~"$job", instance=~"$instance", vs=~"$bigip_virtual_server"}[$__rate_interval])',
+      datasource=promDatasource,
+      legendFormat='{{vs}} - {{partition}} - {{instance}} - received',
+      format='time_series',
+    ),
+    prometheus.target(
+      'rate(bigip_vs_ephemeral_bytes_in{job=~"$job", instance=~"$instance", vs=~"$bigip_virtual_server"}[$__rate_interval])',
+      datasource=promDatasource,
+      legendFormat='{{vs}} - {{partition}} - {{instance}} - ephemeral received',
+      format='time_series',
+    ),
+    prometheus.target(
+      'rate(bigip_vs_clientside_bytes_out{job=~"$job", instance=~"$instance", vs=~"$bigip_virtual_server"}[$__rate_interval])',
+      datasource=promDatasource,
+      legendFormat='{{vs}} - {{partition}} - {{instance}} - sent',
+    ),
+    prometheus.target(
+      'rate(bigip_vs_ephemeral_bytes_out{job=~"$job", instance=~"$instance", vs=~"$bigip_virtual_server"}[$__rate_interval])',
+      datasource=promDatasource,
+      legendFormat='{{vs}} - {{partition}} - {{instance}} - ephemeral sent',
+    ),
+  ],
+  type: 'timeseries',
+  title: 'Traffic',
+  description: 'The rate of data sent and received from clients by the virtual server.',
+  fieldConfig: {
+    defaults: {
+      color: {
+        mode: 'palette-classic',
+      },
+      custom: {
+        axisCenteredZero: false,
+        axisColorMode: 'text',
+        axisLabel: '',
+        axisPlacement: 'auto',
+        barAlignment: 0,
+        drawStyle: 'line',
+        fillOpacity: 30,
+        gradientMode: 'none',
+        hideFrom: {
+          legend: false,
+          tooltip: false,
+          viz: false,
+        },
+        insertNulls: false,
+        lineInterpolation: 'smooth',
+        lineWidth: 2,
+        pointSize: 5,
+        scaleDistribution: {
+          type: 'linear',
+        },
+        showPoints: 'never',
+        spanNulls: false,
+        stacking: {
+          group: 'A',
+          mode: 'none',
+        },
+        thresholdsStyle: {
+          mode: 'off',
+        },
+      },
+      mappings: [],
+      thresholds: {
+        mode: 'absolute',
+        steps: [
+          {
+            color: 'green',
+            value: null,
+          },
+        ],
+      },
+      unit: 'Bps',
+    },
+    overrides: [],
+  },
+  options: {
+    legend: {
+      calcs: [
+        'min',
+        'mean',
+        'max',
+      ],
+      displayMode: 'table',
+      placement: 'right',
+      showLegend: true,
+    },
+    tooltip: {
+      mode: 'multi',
+      sort: 'desc',
+    },
+  },
+};
+
+local packets__intervalPanel = {
```

Comment:
This absolutely needs to be split into at least two panels for `inbound` and `outbound`, possibly even with four if you separate out ephemerals.

Reply:
updated splitting ephemeral too